browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,349 @@ export default class Browse extends React.Component {
     }
 
     if (web.LoggedIn()) {
-      if (!(used === 0 && free === 0)) {
-        storageUsageDetails = <div className="feh-usage">
-          <div className="fehu-chart">
-            <div style={ { width: usedPercent } }></div>
-          </div>
-          <ul>
-            <li>
-              <span>Used: </span>
-              { humanize.filesize(total - free) }
-            </li>
-            <li className="pull-right">
-              <span>Free: </span>
-              { humanize.filesize(total - used) }
-            </li>
-          </ul>
-        </div>
-      }
-
-
+      storageUsageDetails = <div className="browser-status">
+                              <div className="browser-status__storage">
+                                <small>{ humanize.filesize(total - free) } of { humanize.filesize(total) } Used</small>
+                                <div className="browser-status__chart">
+                                  <div style={ { width: usedPercent } }></div>
+                                </div>
+                              </div>
+                            </div>
     }
 
     let createButton = ''
     if (web.LoggedIn()) {
-      createButton = <Dropdown dropup className="feb-actions" id="fe-action-toggle">
-                       <Dropdown.Toggle noCaret className="feba-toggle">
-                         <span><i className="fa fa-plus"></i></span>
+      createButton = <Dropdown dropup className="create-new" id="dropdown-create-new">
+                       <Dropdown.Toggle noCaret className="create-new__toggle">
```

Comment:
This is not centered correctly for me, running OSX 10.11.6, firefox 54.0.1. *[from old pr](https://github.com/minio/minio/pull/4697#discussion_r128386939)*

![screen shot 2017-07-25 at 4 45 39 pm](https://user-images.githubusercontent.com/3038254/28598630-c6a7cc8e-7159-11e7-976e-b64a4bcdddf0.png)